CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ OPTION(WITH_DRM_ZCOPY "enable drm zero-copy support" ON)
 OPTION(WITH_KMS_ZCOPY "enable kms zero-copy support" ON)
 OPTION(WITH_DMABUF_ZCOPY "enable dmabuf zero-copy support" ON)
 OPTION(WITH_WAYLAND "build with wayland backend" ON)
+OPTION(WITH_WAYLAND_PRESENTATION_API "deliver callbacks in more synchronous way using presntation time protocol" OFF)
```

Comment:
"presentation"

Also, do we need this option at all? Why don't always use presentation API?

Reply:
With this option, we have fewer scores on weston-weston setups, so I decided to add this as an option.

Reply:
Ah, makes sense

Reply:
typo fixed